dispatcher.js

```diff
@@ -53,6 +54,12 @@ var Dispatcher = (function(){
     Dispatcher.run(window.App, meta.attr("content"));
   };
 
+  Dispatcher.isRoute = function(route) {
```

Comment:
isRoute is horrible name for a method that throws an exception when it fails. Better go with `Dataclips.assertRoute`.